### ISSUE TITLE
Midterm project - ZIga

### ISSUE THREAD

**Issue by Ziga-K** [Ironhack-Data-0621-Remote/mid-bootcamp-project#5]
https://github.com/Terminal-zkolar/Ironhack-Mid-term-Project-Ziga-

**Comment by Fominayasg** [comment]
# Hi Zyga 😁 !!

Let's have a look at your project!! :rocket: 

## Repo organization
First of all I have to say that I rally apreciate the organization of you rrepository in folders. It makes it look more proffesional.

I see that you have [DS_Store files](https://en.wikipedia.org/wiki/.DS_Store) in you repo (This are temporary files from macOS that doesn't add any value to your repo). It's better if you include them in the `.gitignore` so they don't get uploaded to github. It is important that you create the `.gitignore` file before commiting any other changes, otherwise the unliked flies will be uploaded if you use the command `git add .`.

In Python folder I strongly recommend to numerate the jupyters or to write in the file name what is it inside, like data cleaning, feature extraction or model comparing.

## README
Your readme  is very complete but I would'nt include the detailed steps from notebook. If someone wants to know it they can open the file to read it.

In the libraries used, you can put sklearn only once, without specifying all the models with its links. Remember that the readme is just an introductory view to your repo.

## SQL

Really good job here! Congrats!!

The only thing I can say is that in question 10, there is a huge disparity in the number of houses for a given condition, so it is difficult to establish a correlation between the condition of the house and the grade'.

In question 11 and some other you can set more than one value for a column like this:
```sql
where bedrooms in (3, 4)
```
instead using `or` so you write a little less, but your solution is perfectly fine too.

## EDA & Data Cleaning

I like the way you treated the data but in the visualization part I miss some explanation of the plot. It doesn't have to be very long but you could write one short sentence per plot putting into words what the plot means.

## ML
really liked the precise conclusions and that you included a "future work" part :heart_eyes:.


## Tableau

Very good job here too, the only thing for me is that in question 2 the plots appear a little messy, maybe you can split it into more views or make all of them of te same size. 


In general you did a very good job Ziga. ¡Thanks for your work!
